<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>Analysis of Cancer Genomes by bioinformatics-core-shared-training</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">
    <script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-63148050-6', 'auto');
  ga('send', 'pageview');

</script>
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">Analysis of Cancer Genomes</h1>
      <h2 class="project-tagline">Cancer Research Uk Bioinformatics Summer School: Cambridge, 25th - 29th July 2016</h2>
      <a href="https://github.com/bioinformatics-core-shared-training/cruk-summer-school-2016" class="btn">View on GitHub</a>
      <a href="https://github.com/bioinformatics-core-shared-training/cruk-summer-school-2016/zipball/master" class="btn">Download .zip</a>
      <a href="https://github.com/bioinformatics-core-shared-training/cruk-summer-school-2016/tarball/master" class="btn">Download .tar.gz</a>
    </section>

    <section class="main-content">
      <p><img src="https://raw.githubusercontent.com/bioinformatics-core-shared-training/cruk-bioinf-sschool/gh-pages/img/cruk-cambridge-institute.jpg" alt=""></p
      <p><img align="center" src="https://raw.githubusercontent.com/bioinformatics-core-shared-training/cruk-summer-school-2016/gh-pages/IMG_20160729_123614_171.jpg" alt=""/></p>

<h1>
<a id="description" class="anchor" href="#description" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Description</h1>

<p>High-throughput technologies such as next generation sequencing (NGS) can routinely produce massive amounts of data. These technologies allow us to describe all variants in a genome and international collaborative efforts such as The Cancer Genome Atlas (TCGA) and International Cancer Genome Consortium (ICGC) have begun to catalogue and release data on genomic variation in a variety of cancer types.</p>

<p>However, such datasets pose new challenges in the way the data have to be analyzed, annotated and interpreted which are not trivial and are daunting to the wet-lab biologist. This course covers state-of-the-art and best-practice tools for the analysis of cancer genomes. We describe, and give hands-on experience of, the entire analysis workflow from raw data generated by a sequencing machine to deriving variant calls (SNVs, copy-number and structural variants) that are ready for downstream analysis and interpretation</p>

<h1>
<a id="instructors" class="anchor" href="#instructors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Instructors</h1>

<ul>
<li><a target="_blank" href="https://github.com/markdunning">Mark Dunning</a></li>
<li><a target="_blank" href="http://www.andrewlynch.co.uk/">Andy Lynch</a></li>
<li><a target="_blank" href="http://gmacintyre.com/">Geoff Macintyre</a></li>
<li><a target="_blank" href="http://www.cambridgecancercentre.org.uk/users/oscarrueda">Oscar Rueda</a></li>
<li><a target="_blank" href="http://www.cruk.cam.ac.uk/core-facilities/bioinformatics-core">Matt Eldridge</a></li>
</ul>

<p>We are very grateful for the support of the <a href="http://bioinfotraining.bio.cam.ac.uk/about">Bioinformatics Training</a> facility at the University of Cambridge</p>

<h1>
<a id="aims" class="anchor" href="#aims" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Aims</h1>

<ul>
<li>Appreciation of the nature and scale of NGS data and the requirement for sophisticated computational methods</li>
<li>Describe theory behind current methods for calling SNVs and copy-number changes from NGS data, and their outputs</li>
<li>Encourage exploration of NGS data using interactive tools such the Integrative Genomics Viewer (IGV)</li>
<li>Increase awareness of existing cancer cohorts and how they can be exploited</li>
</ul>

<h1>
<a id="objectives" class="anchor" href="#objectives" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Objectives</h1>

<ul>
<li>Understand the main file formats used for NGS analysis (bam, vcf, bed etc), what is included in each file and appropriate tools for manipulating each file</li>
<li>Know the metrics and tools that can be used to assess if a given sequencing run is adequate quality for analysis</li>
<li>Understand the concepts and challenges involved in calling SNVs from whole-genome data</li>
<li>Given a set of called SNVs, be able to 
i) assess quantitively and qualitatively which calls might be "real" or not 
ii) assess which calls might be biologically-meaningful and warrant further investigation</li>
<li>Know how to access TCGA and ICGC data and how they can inform other studies</li>
</ul>

<h1>
<a id="venue" class="anchor" href="#venue" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Venue</h1>

<ul>
<li>Craik-Marshall room, Department of Genetics</li>
</ul>

<h1>
<a id="accommodation" class="anchor" href="#accommodation" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Accommodation</h1>

<p>If required, free bed and breakfast accommodation wll be provided for attendees in Downing College - close to the course's location. Please let us know on the registration form if you need accommodation and when you plan to check-in and check-out.</p>

<h1>
<a id="timetable" class="anchor" href="#timetable" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Timetable</h1>

<h2>
<a id="day-one-0930---1700" class="anchor" href="#day-one-0930---1700" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Day One 09:30 - 17:00</h2>

<li> 09:30 - 12:30 Course Introductions and R Recap:-
<ul><a target ="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day1/Session1a-intro.html">Course Introduction</a></ul>
<ul><a target ="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day1/Session1b-intro.html">R Recap</a></ul>
<ul><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day1/Session2a-rawReads.html">Introduction to sequencing data</a></ul>
</li>
<p><em>12:30 - 13:30 LUNCH (provided)</em></p>

<li>13:30 - 14:30 Understanding raw sequencing reads:-
<ul><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day1/Session2b-rawReads.html">Hands-on fastq Practical</a></ul>
<ul><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day1/Session2c-rawReads.html">QA of reads</a></ul>
</li>

<li>14:30 - 17:00 Understanding aligned reads:-
<ul><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day1/Session3a-alignedReads.html">Introduction</a></ul>
<ul><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day1/Session3b-alignedReads.html">Introduction to IGV</a></ul>
<ul><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day1/Session3c-alignedReads.html">Hands-on with bam files Practical</a></ul>
<h2>

<a id="day-two-0930---1700" class="anchor" href="#day-two-0930---1700" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Day Two 09:30 - 17:00</h2>

<li><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day2/Lecture1-TuesdayMorning.pdf">09:30 - 10:30 SNV Introduction and calling germline SNVs (Lecture)</a></li>

<li><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day2/work-with-vcf.html">10:30 - 12:30 Calling germline variants and working with VCF files</a></li>

<p><em>12:30 - 13:30 LUNCH (provided)</em></p>

<li>13:30 - 17:00 Copy-Number analysis
<li><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day2/CopyNumberLecture.pdf">Lecture</a></li>
<li><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day2/CopyNumber_DNASEQ-Prac.pdf">Practical</a></li>

</li>


<h2>
<a id="day-three-0930---1700" class="anchor" href="#day-three-0930---1700" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Day Three 09:30 - 17:00</h2>

<li><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day3/Lecture2-WednesdayMorning.pdf">09:30 - 10:00 Somatic SNV calling (Lecture) </a></li>

<li><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day3/SNV_filtering.html">10:00 - 10:45 Filtering SNVs (Lecture)</a> </li>

<li><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day3/assessing-variants.html">10:45 - 12:30 Visualising and Assessing Somatic SNVs  </a></li>

<p><em>12:30 - 13:30 LUNCH (provided)</em></p>

<li><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day3/anno-n-filt.html">13:30 - 16:00 Annotating, Filtering and Prioritising SNVs</a> </li>
<li>16:00 - 17:00 Further issues and considerations
<li><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day3/Lecture3-WednesdayAfternoon.pdf">Lecture</a></li>
<li><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day3/SignaturePractical.html">Practical</a></li>
</li>

<h2>
<a id="day-four-0930---1700" class="anchor" href="#day-four-0930---1700" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Day Four 09:30 - 17:00</h2>
<p><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day4/Day4.html">Index</a></p>
<p><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day4/Day4.Rmd">Solutions to Exercises</a></p>
<li>09:30 - 10:30 Introduction to Structural Variants (SV) and methods for calling SVs</li>
<li>10:30 - 12:30 Visualising SVs </li>
<li><em>12:30 - 13:30 LUNCH (provided)</em></li>
<li>13:30 - 14:30 Intersecting and filter SVs</li>
<li>14:30 - 16:00 Understanding and annotating SVs</li>
<li>16:00 - 17:00 Complex rearrangements and detecting "chromothripsis"</li>
<p><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day4/docker_demo.html">Bonus: Docker Demo</a></p>
<p>WORKSHOP DINNER</p>

<h2>
<a id="day-five-12-day-0930---1230" class="anchor" href="#day-five-12-day-0930---1230" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Day Five (1/2 day) 09:30 - 12:30</h2>
<p><a target="_blank" href="https://rawgit.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/Day5/multisample.pdf">Slides</a></p>

<li> Dealing with large collections of Genomes</li>
<li> Obtaining TCGA data</li>
<li> Mutational signatures</li>
<li> Integrating different data types</li>
<p><em>12:30 - 13:30 LUNCH (provided)</em></li>


<h1>
<a id="software" class="anchor" href="#software" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Software</h1>
</h1>
<p>Practical exercises can be completed in either RStudio, or Docker</p>

<h2>RStudio instructions</h2>

<li>Download and install the latest version of RStudio for your operating system: <a href="https://www.rstudio.com/products/rstudio/download2/">LINK</a> </li>
<li> Once RStudio is loaded, Install the R packages required for the course by typing the command:-</li>
<code>
  source("https://raw.githubusercontent.com/bioinformatics-core-shared-training/cruk-summer-school-2016/master/installBioCPkgs.R")
</code>

<h2>CGPbox instructions</h2>
The Wellcome Trust Sanger Institute have made their cancer genome analysis pipeline available as a Docker container; <a target="_blank" href="https://github.com/cancerit/cgpbox">cgpbox</a>
<li>Install Docker for your operating system. <a target="_blank" href="https://docs.docker.com/docker-for-mac/">Docker for Mac</a> requires OS X 10.10.3 Yosemite or newer. <a target="_blank" href="https://docs.docker.com/docker-for-windows/">Docker for Windows</a> requires Windows 10.</li>
<li>If you do not have Mac OSX 10.10.3 or Windows 10, you will have to install <a target="_blank" href="https://docs.docker.com/toolbox/overview/">Docker Toolbox</a></li>
<li>Follow the installation instructions</li>
<li>Once docker is sucessfully running, you can get the following at the command-line:-</li>
<code>docker pull quay.io/wtsicgp/cgp_in_a_box</code>

<h2>CRUK Docker instructions</h2>
<li>Install docker as above</li>
<li>Download the latest version of the docker container with:-</li>
<code>docker pull markdunning/summer-school2016</code>


      <footer class="site-footer">
        <span class="site-footer-owner"><a href="https://github.com/bioinformatics-core-shared-training/cruk-summer-school-2016">Analysis of Cancer Genomes</a> is maintained by <a href="https://github.com/bioinformatics-core-shared-training">bioinformatics-core-shared-training</a>.</span>

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
